smarts/core/smarts.py

```diff
@@ -258,7 +258,7 @@ def _step(self, agent_actions):
         extras = dict(scores=scores)
         return observations, rewards, dones, extras
 
-    def reset(self, scenario: Scenario):
+    def reset(self, scenario: Scenario, start_time=0):
```

Comment:
Adding this option to the `Scenario` reset has implications on all of the providers and co-simulators. Have you checked the implications on the SUMO simulation?

Reply:
Hmm, I need to double check, now this `start_time` will not affect SUMO.

Reply:
https://github.com/huawei-noah/SMARTS/pull/163/commits/9ac9db0541bf52d35d39cefb4e9488317b74867e makes SUMO's cumulative sim time aligned with elapsed_sim_time